.github/workflows/tasklist-migrate-elasticsearch-data.yml

```diff
@@ -64,12 +64,12 @@ jobs:
       # Build: maven artifacts
       - name: Build backend
         run: |
-          mvn -f tasklist -B -DskipTests -P skipFrontendBuild clean install
+          mvn -B -DskipTests -DskipChecks -T1C -P skipFrontendBuild clean install
```

Comment:
❓ Is it intended to build here to complete repo/dist?

Reply:
yes, here we are testing some code that depends on Zeebe libraries
we need to make sure that the snapshot version of Zeebe matches the same level of code that is being tested.